packages/x-charts/src/internals/plugins/featurePlugins/useChartCartesianAxis/defaultizeAxis.ts

```diff
@@ -33,17 +38,23 @@ export function defaultizeXAxis(
     const defaultHeight =
       DEFAULT_AXIS_SIZE_HEIGHT + (axisConfig.label ? AXIS_LABEL_DEFAULT_HEIGHT : 0);
 
+    const id = axisConfig.id ?? `defaultized-x-axis-${index}`;
     const sharedConfig = {
-      id: `defaultized-x-axis-${index}`,
       offset: offsets[position],
       ...axisConfig,
+      id,
       position,
       height: axisConfig.height ?? defaultHeight,
+      zoom: defaultizeZoom(axisConfig.zoom, id, 'x'),
     };
 
     // Increment the offset for the next axis
     if (position !== 'none') {
       offsets[position] += sharedConfig.height;
+
+      if (sharedConfig.zoom?.slider?.enabled) {
+        offsets[position] += sharedConfig.zoom?.slider?.size ?? DEFAULT_ZOOM_SLIDER_SIZE;
```

Comment:
The zoom is now undefined or the one with default
```suggestion
      if (sharedConfig.zoom?.slider.enabled) {
        offsets[position] += sharedConfig.zoom?.slider.size;
```